thorbanks/utils.py

```diff
@@ -173,8 +175,7 @@ def verify_signature(request, bank_name, signature, auth=False, response=False):
 
     digest = request_digest(request, bank_name, auth=auth, response=response)
 
-    hash_algorithms = [hashes.SHA1(), hashes.SHA256()]
-    for hash_algorithm in hash_algorithms:
+    for hash_algorithm in HASH_ALGORITHMS.values():
```

Comment:
maybe we should now start raising an error here if the wrong algorithm is used (e.g. the algorithm is set in settings but the wrong one matches here). 

thoughts?

Reply:
Added additional unhappy path logging. Additionally, separated the hash algorithm variables to sign hash algorithm and verify hash algorithm.